decider-sample-fingerprinting/src/main/java/ca/on/oicr/pde/deciders/SampleFingerprintingDecider.java

```diff
@@ -95,22 +92,6 @@ public ReturnValue init() {
         Log.debug("INIT");
         String[] metaTypes = {TBI_METATYPE, VCF_GZIP_METATYPE, FIN_METATYPE};
         this.setMetaType(Arrays.asList(metaTypes));
-        // Handle .ini file - we accept only memory size allocated to different steps
-        if (options.has("ini-file")) {
-            File file = new File(options.valueOf("ini-file").toString());
-            if (file.exists()) {
-                String iniFile = file.getAbsolutePath();
-                Map<String, String> iniFileMap = new HashMap<String, String>();
-                MapTools.ini2Map(iniFile, iniFileMap);
-                this.customVcfList = iniFileMap.get("input_files");
-                if (!this.customVcfList.isEmpty()) {
-                    Log.warn("Note that list of .vcf.gz files from this .ini will be APPENDED to the list retrieved from metaDB");
-                }
-            } else {
-                Log.error("The given INI file does not exist: " + file.getAbsolutePath());
-                System.exit(1);
-            }
-        }
 
         this.reseqType = new HashMap<String, Map>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)